donuts/tests/test_image.py

```diff
@@ -75,6 +75,24 @@ def test_returns_self(self):
         testvalue = image.trim()
         assert testvalue is image
 
+    def test_manual_region_choosing(self):
```

Comment:
It would be useful to have some tests on non-square images to check for array indexing issues.